### PR TITLE
Added source directory for Django v2.0.2

### PR DESCRIPTION
**Pull request by SaranshVatsa**
Slight changes that had to be made as compared to the v1.9, especially in the urls.py and views.py.

The directory structure is as per the recommended one in the new Django Documentations, with the app directories and main project directory coming at the same level of hierarchy as manage.py.

README has been updated too for the link to Django 2.0.2 and the instructions to run this project (which is, again, slightly different from the previous ones)

### CODE REVIEW

File: src/for_django_2-0/myproject/myproject/settings.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'myapp',
```
Comment (suhailvs) [inline src/for_django_2-0/myproject/myproject/settings.py]: `'myapp.apps.MyappConfig',` seems the default way.

**Comment by axelpale** [comment]
Damn damn damn. This is an important and valuable PR but I absolutely have had no time to validate this. I appreciate your work @SaranshVatsa but as a convention, we need a second opinion, a peer review. I will merge it then.

So, can anyone else ensure that the PR is a working and fulfills its purpose as a minimal file upload example? Maybe @suhailvs ?

**Comment by suhailvs** [comment]
@axelpale  I hope to do that in the weekend.

**Comment by suhailvs** [comment]
@axelpale done. I made a pull request with some minor updates from @SaranshVatsa commit.

**Comment by axelpale** [comment]
Thank you @suhailvs and @SaranshVatsa . My apologies for this PR being neglected for so long.

**Comment by SaranshVatsa** [comment]
No problem. Better late than never :)

**Pull request merged**